parachains/pallets/bridge-transfer/src/impls.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2023 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::{Config, Pallet};
+use frame_support::traits::{Contains, ContainsPair};
+use xcm::prelude::*;
+use xcm_builder::ExporterFor;
+
+/// `ExporterFor` implementation to check if we can transfer anything to `NetworkId`
+impl<T: Config> ExporterFor for Pallet<T> {
+	fn exporter_for(
+		network: &NetworkId,
+		_remote_location: &InteriorMultiLocation,
+		_message: &Xcm<()>,
+	) -> Option<(MultiLocation, Option<MultiAsset>)> {
+		Self::allowed_exporters(network)
+			.map(|bridge_config| (bridge_config.bridge_location, bridge_config.bridge_location_fee))
```

Comment:
I see that the `UnpaidRemoteExporter` throws an error if `maybe_payment` is not `None`: https://github.com/paritytech/polkadot/blob/df61c0bf0834f650e79ddd1a74dea34081ff2b46/xcm/xcm-builder/src/universal_exports.rs#L156

So `bridge_config.bridge_location_fee` seems misguiding. If someone sets it to `Some(fee)` if will (unexpectedly) actually lead to failure. Would it make sense to remove it ?